pallets/msa/src/lib.rs

```diff
@@ -556,6 +571,51 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Retire a MSA
+		///
+		/// ### Events
+		/// - Deposits [`MsaRetired`](Event::MsaRetired) when MSA is retired
+		///
+		/// ### Errors
+		///
+		/// - Returns [`NoKeyExists`](Error::NoKeyExists) if `delegator` does not have an MSA key.
+		/// - Returns [`MoreThanOneKeyExists`](Error::MoreThanOneKeyExists) if the MSA has more than one account key.
+		/// - Returns [`RegisteredProviderCannotBeRetired`](Error::RegisteredProviderCannotBeRetired) if the MSA id is a registered provider
+
+		#[pallet::weight((T::WeightInfo::retire_msa(), DispatchClass::Normal, Pays::No))]
```

Comment:
Do we need to update or create a SignedExtension for this? If so perhaps make this a  Pays::Yes until we add it.

Reply:
I will make it Pays::Yes